packages/react-checkbox/src/Checkbox.stories.tsx

```diff
@@ -1,12 +1,23 @@
 import * as React from 'react';
 import { Checkbox } from './index';
 
-// TODO add 'react-checkbox' to the list for withFluentProvider in react-examples/.storybook/preview.js
+export const CheckboxVariations = () => (
+  <div style={{ display: 'flex', gap: '14px', flexDirection: 'column' }}>
+    <Checkbox label="Simple Checkbox" />
```

Comment:
I thought that we agreed to use children as label ?

```suggestion
    <Checkbox>Simple Checkbox</Checkbox>
```

Reply:
We did not do that because we cannot add props to the label, but we are rethinking how the root slot works and might add this in a future PR. It also depends on what's the outcome of #18804, #18983.

Reply:
For future reference, this PR (#19225) introduces back the agreed children as label by using label as the root of Checkbox.